Resources/views/Form/fields.html.twig

```diff
@@ -1,8 +1,17 @@
 {% block cmf_media_image_widget %}
-    {{ form_widget(form) }}
     {% block cmf_media_image_widget_preview %}
         {% if form.vars.data and form.vars.data.id is defined and form.vars.data.id %}
                 <img src="{{ cmf_media_display_url(form.vars.data, { imagine_filter : imagine_filter }) }}" alt="" {% if not imagine_filter %}width="100"{% endif %} />
         {% endif %}
     {% endblock %}
+    {{ form_widget(form) }}
```

Comment:
OK I will revert it to the original position to avoid breaking compatibility. 
Anyway, to me it makes more sense having the preview of the uploaded image before the upload button (for replacing it). I could possibly propose this in a PR for a future version of the bundle.